### DISCUSSION TITLE
eslint-plugin-simple-import-sort

### DISCUSSION THREAD

**Discussion by Boshen** [oxc-project/oxc#7197] [Feature Request]
https://github.com/lydell/eslint-plugin-simple-import-sort

**Comment by pumano** [comment]
I really love this and use it, but its good if it will be implemented with fixers) (which is not trivial task)

**Reply by camc314** [reply]
i think easiest way to implement this (i'm guesssing i haven't looked at the src code), is to build the fixed version and compare it to the original text.
only problem will probably be comments

**Reply by pumano** [reply]
`import/order` looks the same, but `simple import sort` is more advanced, and provide possibility to sort custom things (for example local namespaces @companyname/package) too

**Comment by Boshen** [comment]
https://github.com/oxc-project/oxc/issues/14253
